recipes/carpedeam/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "1.0.0" %}
+{% set sha256 = "92deec3d670810f8433c0345758588b47fa460349e61ce8aa549374f5cf4a812" %}
+
+package:
+  name: carpedeam 
+  version: {{ version|replace("-", ".")  }}
```

Comment:
There is no need of `| replace` since the version does not have any `-`s. But it does not do any harm either.